presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -291,11 +309,39 @@ private void testUnsupportedDataType(String databaseDataType)
                 Function.identity());
     }
 
+    public static DataType<String> jsonDataType()
+    {
+        return DataType.dataType(
+                "json",
+                JSON,
+                value -> format("JSON'%s'", value),
+                Function.identity());
+    }
+
+    @Test
+    public void jsonDataTypeTest()
```

Comment:
move it just after the existing `testTimestamp`

(i think the name need to be revisited though)

Reply:
done